test/common/voldemort/ServerTestUtils.java

```diff
@@ -172,6 +179,22 @@ public static AbstractSocketService getSocketService(boolean useNio,
         return socketService;
     }
 
+    public static AbstractSocketService getSocketService(boolean useNio,
+                                                         RequestHandlerFactory requestHandlerFactory,
+                                                         int port,
+                                                         int coreConnections,
+                                                         int maxConnections,
+                                                         int bufferSize) {
+        return getSocketService(useNio,
+                                requestHandlerFactory,
+                                port,
+                                coreConnections,
+                                maxConnections,
+                                bufferSize,
+                                TimeUnit.MILLISECONDS.convert(3, TimeUnit.MINUTES));
```

Comment:
I think you already have this default value set in ```VoldemortConfig```, right? You're hard-coding it in two other places. Can we use ```VoldemortConfig.getNioSelectorMaxHeartBeatTimeMs()```?